crates/catalog/sql/src/catalog.rs

```diff
@@ -141,21 +142,30 @@ impl SqlCatalog {
     }
 
     /// SQLX Any does not implement PostgresSQL bindings, so we have to do this.
-    pub async fn execute_statement(
-        &self,
-        query: &String,
-        args: Vec<Option<&String>>,
-    ) -> Result<Vec<AnyRow>> {
-        let query_with_placeholders: Cow<str> =
-            if self.sql_bind_style == SqlBindStyle::DollarNumeric {
-                let mut query = query.clone();
-                for i in 0..args.len() {
-                    query = query.replacen("?", &format!("${}", i + 1), 1);
-                }
-                Cow::Owned(query)
-            } else {
-                Cow::Borrowed(query)
-            };
+    fn replace_placeholders(&self, query: &str) -> String {
+        match self.sql_bind_style {
```

Comment:
@Xuanwo @liurenjie1024 thoughts on this? Keen to close this PR out.

Reply:
the dollar-numeric bind style is postgres specific - the ? syntax is for dbs like mysql, sqlite. I am pretty sure only sqlite supports both styles.

Reply:
Ok - I've added some comments to the config to explain the parameter's options. It might be worth a rethink when I do the Table functionality in this coming week. Maybe as an Option that defaults to looking at the db type if not provided.